packages/endpoints-organizations-registrar/src/entities/signatories/orchestrators/send-reminders.js

```diff
@@ -51,6 +51,34 @@ const sendReminder = async (
       return;
     }
 
+    const isMaxReminderCount = flow(size, (n) =>
```

Comment:
very functional but the native code version seems much easier to understand

```
if (size(signatoryStatusDoc.events) >= config.signatoryMaxReminderCount) {
   ...
}
```

Reply:
done, [92c6d9c](https://github.com/velocitycareerlabs/velocitycore/pull/179/commits/92c6d9c5fc0f8ee03fd0e5726b9d50222b5d8862)